include/l3.h

```diff
@@ -143,6 +162,15 @@ void l3_log_mmap(const char *msg, const uint64_t arg1, const uint64_t arg2,
                  const uint32_t loc);
 #endif  // L3_LOC_ENABLED
 
+/**
+ * l3_log_fprintf() - Log a msg to a file using fprintf().
+ */
+static inline void
+l3_log_fprintf(const char *msg, const uint64_t arg1, const uint64_t arg2)
+{
+    fprintf(l3_log_fh, msg, arg1, arg2);
```

Comment:
Just a note that this is fine for benchmarking purposes, but won't work on all architectures (namely i386) if the strings format specifiers aren't 64-bit (e.g. `%c` or `%d`).

Reply:
The idea, and expectation, here is that the printf() format-specifier validation I added (in debug mode, under SHA f280675) is supposed to come into play.

If the specific platform requires a different format-specifiers, code author is supposed to invoke it with those, othewise, this will fail in debug builds, defined in `l3.h`:

```
 89             } else if (0) {                                             \
 90                 printf((msg), (arg1), (arg2));                          \
```

I agree that now as the `fprintf()` interfaces are going public, we have to do a minor revisit on usability. Let's hold-off on that till we get these changes in first.